main_training.py

```diff
@@ -344,7 +421,11 @@ def fsdp_main():
         device_id=torch.cuda.current_device(),
         forward_prefetch=cfg.forward_prefetch,
         limit_all_gathers=False,
+        param_init_fn=my_init_fn
     )
+    print_memory_summary("vit","cuda")
+
+    time.sleep(10)
```

Comment:
same as above - assuming this is just a review nicety for the user to review w/o console blazing by?